trivy-image-scan/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Trivy Image Scan
+description: 'Trivy Image Scan'
+inputs:
+  image-ref:
+    description: 'image reference'
```

Comment:
Could we split image name and tag inputs so we can default generate the tag in here? Just trying to reduce code copy-paste.

Reply:
done.